src/bootstrap/config.rs

```diff
@@ -494,6 +494,7 @@ struct Rust {
     llvm_libunwind: Option<String>,
     control_flow_guard: Option<bool>,
     new_symbol_mangling: Option<bool>,
+    download_stage1: Option<bool>,
```

Comment:
This should be added to `config.toml.example` too.

Reply:
I also added a profile for rustdoc that turns this on by default.

Reply:
I reverted this when I reverted adding `config.tools.toml`. @Mark-Simulacrum do you prefer that I add it here or wait for a follow-up?

Reply:
New options should certainly be added to .example, but I don't think this PR should add it to any defaults until it gets some more testing.